components/Address.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Address": {
+    "name": "address",
+    "schema": {
+      "$ref": "./Address.json#/address"
+    }
+  },
+  "Addresses": {
+    "name": "addresses",
+    "schema": {"$ref": "./Address.json#/addresses"}
+  },
+  "orNull": {
+    "name": "address or null",
+    "schema": {
+      "$ref": "./Address.json#/addressOrNull"
+    }
+  },
+  "Miner": {
+    "name": "coinbase",
+    "summary": "address that received this block's mining rewards",
+    "schema": {
+      "$ref": "./Address.json#/miner"
+    }
+  },
+  "miner": {
+    "title": "coinbase",
+    "summary": "address that received this block's mining rewards",
+    "$ref": "./Address.json#/address"
+  },
+  "addressOrNull": {
+    "title": "address or null",
+    "oneOf": [
+      {
+        "$ref": "./Address.json#/address"
+      },
+      {
+        "$ref": "./Null.json#/null"
+      }
+    ]
+  },
+  "addresses": {
```

Comment:
What's the purpose of having this twice? I think it has to do with generating the object-specific docs after the RPC methods?

Reply:
Yeah so basically open RPC requires you not repeat "schema" and if you want to nest schemas within schemas then you need to keep them as schema not open RPC components.

It's sort of confusing but this basically just allows for us to more easily build out components that reuse the same internal schemas.

The primary purpose of this is to minimize repeats because not reusing will be more challenging to maintain.

Reply:
I don't think I follow. If you look at the spec in the `main` branch here, I didn't have any duplication and have not had any issues building out the components?